### PR TITLE
Implement history backend

### PR DESCRIPTION
**Pull request by drake25122000**

### CODE REVIEW

File: history-service/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+router.get("/historyList/:username", verifyToken, getHistoryList);
+router.get("/:username/:roomId", verifyToken, getHistory);
+router.post("/:username", verifyToken, createHistory);
+
+app.use("/api/history", router).all((_, res) => {
+  res.setHeader("content-type", "application/json");
+  res.setHeader("Access-Control-Allow-Origin", "*");
+});
+
+app.listen(8005, () => console.log("history-service listening on port 8000"));
```
Comment (jzwoo) [inline history-service/index.js:31]: listening on port 8005

File: history-service/controller/history-controller.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            }
+        } else {
+            return res
+                .status(400)
+                .json({ message: "Username is missing!" });
+        }
+    } catch (err) {
+        return res.status(500).json({ message: "Database error!" });
+    }
+}
```
Comment (jzwoo) [inline history-service/controller/history-controller.js:79]: new line

File: history-service/model/repository.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import HistoryModel from "./history-model.js";
+import "dotenv/config";
+
+//Set up mongoose connection
+import mongoose from "mongoose";
+
+let mongoDB =
+  process.env.ENV == "PROD"
```
Comment (jzwoo) [inline history-service/model/repository.js:8]: i think its '==='

**Review by jzwoo — CHANGES_REQUESTED** [review]

**Review by erinmayg — APPROVED** [review]
LGTM

**Pull request closed without merging**